src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -147,6 +147,8 @@ def test_aks_create_default(self, resource_group, resource_group_location):
             self.check('agentPoolProfiles[0].nodeLabels.label2', 'value2'),
             self.check('agentPoolProfiles[0].tags.tag1', 'tv1'),
             self.check('agentPoolProfiles[0].tags.tag2', 'tv2'),
+            self.check('agentPoolProfiles[0].maxPods', 110),
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=78530549&view=results) to validate the change.

Reply:
test passed